src/views/HomeView.vue

```diff
@@ -9,17 +9,43 @@
           Find your favorite Pokemon!
         </p>
       </header>
+      <form class="flex space-x-2 items-center justify-between px-6 py-4">
+        <label
+          for="pokemon_name"
+          class="block flex-1 relative"
+        >
+          <div class="absolute inset-y-0 left-0 pl-3 flex items-center pointer-events-none">
+            <SearchIcon class="w-5 h-5" />
+          </div>
+          <input
+            type="text"
+            name="search"
+            placeholder="Pokemon Name"
+            aria-label="Search Pokemon Name"
+            class="pr-3 pl-10 py-2 bg-emerald-100 placeholder-gray-500 text-black rounded-lg border-none focus:ring-emerald-500 focus:ring-2 w-full"
+          >
+        </label>
```

Comment:
## Minor

Hello @sheilaazhar, #IMO if you nest the `<input>` directly inside the `<label>` in which case the `for` attribute is not needed because the association is implicit. Except, if you explicitly associate an `<label>` element with an `<input>` element. So, you should add `for` and `id` attributes. ✌🏽 

ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
Hello @doohanas thankyou for the insight. If I delete the `for` attribute, I got an error that "Form label must have an associated control". So I keep use `for` and add `id` attributes.